src/docsfy/models.py

```
@@ -116,3 +117,12 @@ class DocPlan(BaseModel):
     tagline: str = ""
     navigation: list[NavGroup] = Field(default_factory=list)
     version: str | None = None
+
+
+def is_uuid(value: str) -> bool:
+    """Check if a string is a valid UUID."""
+    try:
+        _uuid_mod.UUID(value)
+        return True
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)